akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -416,6 +420,19 @@ private[akka] object Running {
 
       replication.clearContext()
 
+      val sideEffects = ackToOnPersisted match {
+        case None => Nil
+        case Some(ref) =>
+          SideEffect { (_: S) =>
+            // FIXME do we really have to create this every time?
+            val pid = ReplicationId(
+              setup.persistenceId.entityTypeHint,
+              setup.persistenceId.entityId,
+              event.originReplica).persistenceId
+            ref ! EventConsumed(pid, event.originSequenceNr)
```

Comment:
Do we need this ack message or can we use `Done`? If it's intended to be sent with `ask` the requester should already know the pid and seqNr?

Reply:
Yeah, might not be needed.